libs/utils/executor.py

```diff
@@ -52,40 +161,12 @@ class Executor():
             'thermal-engine'
         ]
     }
-    """Tasks in the system that we can't afford to freeze"""
+    """
+    Dictionary mapping OS name to list of task names that we can't afford to
+    freeze when using freeeze_userspace.
+    """
 
     def __init__(self, test_env, experiments_conf):
-        """
-        Tests Executor
-
-        A tests executor is a module which support the execution of a
-        configured set of experiments. Each experiment is composed by:
-        - a target configuration
-        - a worload to execute
-
-        The executor module can be configured to run a set of workloads (wloads)
-        in each different target configuration of a specified set (confs). These
-        wloads and confs can be specified by the "experiments_conf" input
-        dictionary. Each (workload, conf, iteration) tuple is called an
-        "experiment".
-
-        All the results generated by each experiment will be collected a result
-        folder which is named according to this template:
-            results/<test_id>/<wltype>:<conf>:<wload>/<run_id>
```

Comment:
Yeah, I replaced this with the doc for `Experiment.out_dir` because I didn't think the output directory structure a hard-defined part of `Executor`'s interface (in fact this is already wrong if you set 'results_dir' in your `test_conf`). I can put it back though.